webview/cocoa.py

```diff
@@ -57,6 +57,16 @@ class BrowserDelegate(AppKit.NSObject):
         def webView_contextMenuItemsForElement_defaultMenuItems_(self, webview, element, defaultMenuItems):
             return nil
 
+        # Display a JavaScript alert panel containing the specified message
+        def webView_runJavaScriptAlertPanelWithMessage_initiatedByFrame_\
+                (self, webview, message, frame):
```

Comment:
[This](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) and [this](https://github.com/r0x0r/pywebview/blob/master/webview/cocoa.py#L135).
I usually break lines under 80 chars (I code in the good old Terminal `80x24` 🤓). And `cocoa` is full of these _long_ method names.

But if you insist, I shall rewrite it in a single line. Or is there some other _pythonic_ way?